src/containers/AccountManagement.jsx

```diff
@@ -53,10 +54,30 @@ export default class AccountManagement extends Component {
       })
   }
 
+  validInfos (input) {
+    const { t } = this.context
+    // Check if field is empty or not
+    if (input.target.value !== '') {
```

Comment:
@GoOz I don't know, I am allergic to if/else statements so let @CPatchane decide. And I did not get deeper at this time. ;)